app/pages/sections/order-items/catalogue-solutions/order-item/tiered/patient/ui-tests/withoutSavedData.ui.test.js

```diff
@@ -0,0 +1,144 @@
+    await setState(ClientFunction)(sessionKeys.selectedRecipientName, selectedRecipientNameInSession);
+    await setState(ClientFunction)(sessionKeys.selectedItemId, itemIdInSession);
+    await setState(ClientFunction)(sessionKeys.selectedItemName, itemNameInSession);
+    await setState(ClientFunction)(sessionKeys.selectedPriceId, selectedPriceIdInSession);
+    await setState(ClientFunction)(sessionKeys.catalogueSolutionId, catalogueSolutionIdInSession);
+    await setState(ClientFunction)(sessionKeys.plannedDeliveryDate, deliveryDateInSession);
+    await setState(ClientFunction)(sessionKeys.recipients, recipientsInSession);
+    await setState(ClientFunction)(sessionKeys.selectedRecipients, selectedRecipientsInSession);
+  }
+  if (setup.postRoute) {
+    await setState(ClientFunction)(sessionKeys.orderItemPageData, orderItemPageDataInSession);
+  }
+};
+
+fixture('Catalogue-solutions - flat patient - withoutSavedData')
+  .page('http://localhost:1234/order/some-fake-page')
+  .afterEach(async (t) => {
+    await nockAndErrorCheck(nock, t);
+  });
+
+test('should navigate to catalogue-solutions dashboard page if save button is clicked and data is valid', async (t) => {
+  nock(orderApiUrl)
+    .post('/api/v1/orders/order-id/order-items/batch', [{ ...requestPostBody, quantity: 10 }])
+    .reply(200, {});
+
+  await pageSetup();
+  await t.navigateTo(pageUrl);
+
+  const quantityInput = Selector('[data-test-id="question-quantity"]');
+  const saveButton = Selector('[data-test-id="save-button"] button');
+
+  await t
+    .typeText(quantityInput, '10', { paste: true })
+    .click(saveButton)
+    .expect(getLocation()).eql('http://localhost:1234/order/organisation/order-id/catalogue-solutions');
+});
+
+test('should show text fields as errors with error message when there are BE validation errors', async (t) => {
+  nock(orderApiUrl)
+    .post('/api/v1/orders/order-id/order-items/batch', [{ ...requestPostBody, quantity: 0 }])
+    .reply(400, {
+      errors: {
+        '[0].Quantity': ['QuantityGreaterThanZero'],
+      },
+    });
+
+  await pageSetup();
+  await t.navigateTo(pageUrl);
+
+  const errorSummary = Selector('[data-test-id="error-summary"]');
+  const solutionTableError = Selector('[data-test-id="solution-table-error"]');
+  const quantityInput = Selector('[data-test-id="question-quantity"] input');
+  const saveButton = Selector('[data-test-id="save-button"] button');
+
+  await t
+    .typeText(quantityInput, '0', { paste: true })
+    .click(saveButton);
+
+  await t
+    .expect(errorSummary.exists).ok()
+    .expect(errorSummary.find('li a').count).eql(1)
+    .expect(await extractInnerText(errorSummary.find('li a').nth(0))).eql(content.errorMessages.QuantityGreaterThanZero)
+
+    .expect(solutionTableError.exists).ok()
+    .expect(await extractInnerText(solutionTableError)).contains(content.errorMessages.QuantityGreaterThanZero);
+});
```

Comment:
We don't need to support tiered prices yet so we can remove the code that relates to tiered pricing. However, keep a copy (either in a stash and/or a copy of the files) as we may be able to make use of it later.

Reply:
Removed tiered folder also kept a copy of the files.